content/post/2017-04-27-the-new-and-improved-r-podcast-site.md

```diff
@@ -0,0 +1,27 @@
+
+I was fortunate enough to attend [rstudio::conf](https://www.rstudio.com/conference/) 2017 and that's where I learned of Yihui's new [blogdown](https://github.com/rstudio/blogdown) package that provides a easy and intuitive wrapper to the [Hugo](http://hugodocs.netlify.com/) website generator.  Immediately I did a bit of research and found that there was a Hugo theme called [castanet](https://github.com/mattstratton/castanet) tailored for podcasts!  It almost seemed too good to be true: I could author the site content with my favorite workflow, and also have the special enhancements needed for the podcast media content practically out of the box.  During one of the evenings I told Yihui about my plans to move to blogdown and he was very supportive and happy to help if I encountered any bumps along the way.  It's been quite a journey, and there is more work to be done, but the site is now powered by blogdown!  What follows is a recap of the technical configurations and my future plans for the R-Podcast site.
+
+## A Brief Tour
+
+Before I go too much further I want to thank the author of the castanet theme [Matt Stratton](https://github.com/mattstratton) who hosts the [Arrested DevOps](https://www.arresteddevops.com/) podcast.  Many of the features I tried to build from scratch when using Nikola were already solved by his excellent theme.  The first thing you will see when visiting the R-Podcast site is a partial listing of each episode sorted by most recent.  One of the many nice features of the [castanet](https://github.com/mattstratton/castanet) theme is a lightweight player for each episode that fits nicely with the overall appearance based on bootstrap.  On the side margin there are sections for the various methods of subscribing to the podcast, a collection of my favorite sites related to R, and a simple widget to join a mailing list.  
+
+Each episode is launched by using a custom `episode` archetype, and within blogdown I make a call to `new_content("episode/name_of_episode.md")` and I'm ready to go.  I have plans to create a custom RStudio add-in for authoring new episode posts that mimics the [new post addin](https://github.com/rstudio/blogdown/blob/master/inst/scripts/new_post.R) already part of blogdown, which if nothing else will give me more show content later on!  
```

Comment:
I was thinking about that as well.  I'm not sure if this would be possible but the inputs could be dynamically generated depending on the `kind` selected by the user and the app could scrape the archetype yaml front matter for the option names.  Most of these inputs would look similar to what you have in the current addin with potentially more depending on the archetype. I'd be glad to give this an attempt :thumbsup: